modules/convert/issue.go

```diff
@@ -168,20 +174,64 @@ func ToTrackedTimeList(tl models.TrackedTimeList) api.TrackedTimeList {
 }
 
 // ToLabel converts Label to API format
-func ToLabel(label *models.Label) *api.Label {
-	return &api.Label{
+func ToLabel(label *models.Label, repoCache map[int64]*models.Repository, orgCache map[int64]*models.User) *api.Label {
+	result := &api.Label{
 		ID:          label.ID,
 		Name:        label.Name,
 		Color:       strings.TrimLeft(label.Color, "#"),
 		Description: label.Description,
 	}
+
+	// calculate URL
+	if label.BelongsToRepo() {
+		if repoCache == nil {
+			repoCache = make(map[int64]*models.Repository)
+		}
+		_, ok := repoCache[label.RepoID]
+		if !ok {
+			var err error
+			repoCache[label.RepoID], err = models.GetRepositoryByID(label.RepoID)
+			if err != nil {
+				log.Error("cant load repo of label [%d]: %v", label.ID, err)
+			}
+		}
+
+		if repo := repoCache[label.RepoID]; repo != nil {
+			result.URL = fmt.Sprintf("%s/labels/%d", repo.APIURL(), label.ID)
+		}
+	}
+	if label.BelongsToOrg() {
```

Comment:
As far as I remember labels belong either to a repo or to an org shouldn't this be an `} else if ... {` or even just `} else {` ?